src/Playgrounds/CosmosEndToEnd/CosmosEndToEnd.ApiService/CosmosEndToEnd.ApiService.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
I don't think we should use the word "Playgrounds" and this shouldn't be under the `src` folder - which is for product code.

I'd rather call these "TestProjects", even if they are manual tests.